WordPress/Classes/Services/NotificationSyncMediator.swift

```diff
@@ -22,7 +22,7 @@ let NotificationSyncMediatorDidUpdateNotifications = "NotificationSyncMediatorDi
 
 // MARK: - NotificationSyncMediator
 //
-class NotificationSyncMediator {
+final class NotificationSyncMediator {
```

Comment:
Was this a change suggested by the compiler?

Reply:
No. I wanted to use the static variable using `Self.operationQueue`, instead of `NotificationSyncMediator.operationQueue`, and I don't see any need of extending this class, so I made it final. 😄

Reply:
👍